Task_20/Pages/BasePage.cs

```diff
@@ -0,0 +1,35 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+
+namespace Yandex.Pages
+{
+    public class BasePage
+    {
+        protected IWebDriver driver;
+
+        public BasePage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public bool IsElementVisible(By searchElementBy)
+        {
+            try
+            {
+                bool result = driver.FindElement(searchElementBy).Displayed;
+                return result;
+            }
+            catch (NoSuchElementException)
+            {
+                return false;
+            }
+        }
+
+        public void WaitElement(By element)
```

Comment:
You may improve this method by adding the second parameter (TimeSpan) which will allow you to adjust timeout value during WebDriverWait instances creation. The hardcoded TimeSpan.FromSeconds(10) is simple decision, but not flexible actually.

Reply:
done